heareval/predictions/task_predictions.py

```diff
@@ -254,6 +254,35 @@ def validation_step(self, batch, batch_idx):
     def test_step(self, batch, batch_idx):
         return self._step(batch, batch_idx)
 
+    def log_scores(self, name: str, score_args):
+        """Logs the metric score value for each score defined for the model"""
+        assert hasattr(self, "scores"), "Scores for the model should be defined"
+        end_scores = {}
+        # The first score in the first `self.scores` is the optimization criterion
+        for score in self.scores:
```

Comment:
We probably only need to do this for the *primary* score, otherwise we are wasting time during validation.

https://github.com/neuralaudio/hear-eval-kit/issues/344